libs/core/src/components.d.ts

```diff
@@ -233,39 +233,39 @@ export namespace Components {
          */
         "componentId": string;
         /**
-          * Indicates whether or not the input field is disabled
+          * If true, the input field is disabled.
          */
         "disabled"?: boolean;
         /**
-          * Specifies the error message and provides an error-themed treatment to the field
+          * Specifies the error message and provides an error-themed treatment to the field.
          */
         "errorMessage"?: string;
         /**
-          * Displays a message or hint below the input field
+          * Displays a message or hint below the input field.
          */
         "helperMessage"?: string;
         /**
-          * Indicates whether or not the input field is invalid or throws an error
+          * If true, the input field is invalid.
          */
         "invalid"?: boolean;
         /**
-          * Text to be displayed as the input label
+          * Text to be displayed as the input label.
          */
         "label"?: string;
         /**
-          * Specifies the name. Submitted with the form name/value pair
+          * Specifies the name. Submitted with the form name/value pair.
          */
         "name"?: string;
         /**
-          * Specifies a short hint that describes the expected value of the input field
+          * Specifies a short hint that describes the expected value of the input field.
          */
         "placeholder"?: string;
         /**
-          * Indicates whether or not the input field is readonly
+          * If true, the input field is readonly.
          */
         "readonly"?: boolean;
         /**
-          * Indicates whether or not the input field is required
+          * If true, the input field is required.
```

Comment:
Does this visually apply styles when required? Does it prevent the field from being submitted, therefore stopping form submission?